docs/user/benchmarking.md

```diff
@@ -0,0 +1,89 @@
+# Benchmarking
+
+This guide explains how to write and run garage benchmark scripts.
+
+## Prerequisites
+
+Firstly, you should have installed garage. If not, you can see instructions in the [latest documentation](https://garage.readthedocs.io/en/latest/user/installation.html) to install garage.
+
+Install `google-cloud-storage`:
+
+`pip install google-cloud-storage`
```

Comment:
This has been [covered](https://github.com/rlworkgroup/garage/blob/19f47bba3baf9575d2cce3aa9d5d8af102fea50b/benchmarks/setup.py#L14) in the next step.

Reply:
Got it. I will remove it.